perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -840,9 +863,26 @@ def Prepare(benchmark_spec):
   # Setup common test tools required on the client VM
   vms[0].Install('sysbench05plus')
 
+
+  vms[0].mysql_svc_oltp_tables_count = FLAGS.mysql_svc_oltp_tables_count
```

Comment:
Agreed but we don't pass the benchmark spec to PrepareSysbench or IssueSysbenchCommand..

As per offline conversation will leave as is unless @stfeng has a different opinion.